worker/worker.py

```diff
@@ -125,23 +125,27 @@ def get_rate():
 def heartbeat(worker_info, password, remote):
     global ALIVE, RUN, TASK_ID
     print("Start heartbeat")
-    payload = {"username": worker_info["username"], "password": password}
+    payload = {
+        "username": worker_info["username"],
+        "password": password,
+        "unique_key": worker_info["unique_key"],
+    }
     count = 0
     while ALIVE:
         time.sleep(1)
         count += 1
         if count == 60:
             count = 0
-            print("Send heartbeat... ", end="")
+            print("Send heartbeat for", worker_info["unique_key"], end=" ... ")
```

Comment:
It's a bit hard to read the heartbeat message now, I suggest to indent with two space:

```
Finished game 153 (New-3f6451eff7 vs Base-3f6451eff7): 1-0 {White wins by adjudication}
Score of New-3f6451eff7 vs Base-3f6451eff7: 49 - 49 - 55  [0.500] 153
Started game 156 of 200 (Base-3f6451eff7 vs New-3f6451eff7)
  Send heartbeat for 695867dc-48a1-434d-8269-36edf45ee0f8 ... user00-3cores-16
Finished game 155 (New-3f6451eff7 vs Base-3f6451eff7): 0-1 {Black wins by adjudication}
Score of New-3f6451eff7 vs Base-3f6451eff7: 49 - 50 - 55  [0.497] 154
```

Reply:
Done...